pkg/controller/common/util.go

```diff
@@ -765,7 +783,7 @@ func ValidateRequestedCloneSize(sourceResources corev1.ResourceRequirements, tar
 	targetRequest := targetResources.Requests[corev1.ResourceStorage]
 	// Verify that the target PVC size is equal or larger than the source.
 	if sourceRequest.Value() > targetRequest.Value() {
```

Comment:
maybe if you are already changing stuff here you can use: sourceUsableSpace.Cmp(targetUsableSpace) > 0 like done in the other check instead? I think thats the better way? not a must

Reply:
Prefer to keep this PR on the safe side